cpp/dolfin/fem/DofMap.h

```diff
@@ -230,6 +218,17 @@ class DofMap : public GenericDofMap
   /// Get dofmap array
   Eigen::Map<const Eigen::Array<PetscInt, Eigen::Dynamic, 1>> dof_array() const;
 
+  // FIXME: can this be removed?
```

Comment:
Any reason why `tabulate_local_to_global_dofs` should be eventually removed?
I would like to state my interest in its maintenance since I use this information to construct my preconditioner.

Reply:
It can be easily got from the `IndexMap`.